src/char/hammerdin.py

```diff
@@ -55,7 +56,9 @@ def on_capabilities_discovered(self, capabilities: CharacterCapabilities):
         if capabilities.can_teleport_natively:
             self._do_pre_move = False
         #hammerdin needs to be closer to shenk to reach it with hammers
-        self._pather.offset_node(149, (70, 10))
```

Comment:
Can we just put in constructor instead?

Reply:
that's where we had it to begin with right? i thought you said it was in the wrong place that's why you moved it

Reply:
It turned out it doesn't need to depend on capabilities. But also depending on capabilities is wrong cause capabilities can change  (theoretically) and you only set pather once. So putting it back to constructor is better